### ISSUE TITLE
Core-AAM: Add language regarding aria-setsize with a value of -1

### ISSUE THREAD

**Issue by joanmarie** [w3c/aria#567]
-1 was added as a valid value for `aria-setsize` as a means to indicate that the set size is unknown and should not be calculated by user agents. The Core AAM says nothing about this. And at least one user agent is calculating the value in response to -1, rather than treating -1 as a valid value.

**Comment by klown** [comment]
Something similar is needed for aria-colcount and aria-rowcount when they have their special case value of -1.

**Comment by joanmarie** [comment]
Looking at the [ATK documentation](https://developer.gnome.org/atk/stable/atk-AtkState.html#AtkStateType), it says:

> ATK_STATE_INDETERMINATE: ... In the case of a large data set in which the total number of items in that set is unknown (e.g. 1 of 999+), implementors should expose the currently-known set size (999) along with this state.

So arguably the correct mapping would be to continue doing the calculation, but adding the state. @asurkov: What do you think about doing the same thing for IA2?

**Comment by asurkov** [comment]
@joanmarie I started a new IA2 thread on the issue [1]

[1] https://lists.linuxfoundation.org/pipermail/accessibility-ia2/2017-August/002222.html

**Comment by asurkov** [comment]
So, Jamie suggested to go with groupPosition returning -1, like it.

**Comment by joanmarie** [comment]
@asurkov: Thanks!

Looking more at the language in the Core AAM, it already indicated the value should be exposed; not that a value which was a positive integer was exposed. Thus arguably -1 was already covered.

Having said that, the ATK mappings were missing `STATE_INDETERMINATE` along with what I quoted above from the ATK documentation about exposing the currently-known size, so I've made that change for ATK and left the rest as-is.
